packages/web3-react/src/components/AlephiumConnect.tsx

```diff
@@ -208,14 +194,15 @@ export const AlephiumConnectProvider: React.FC<{
     }
 
     func()
-  }, [lastConnectedAccount, network, addressGroup, keyType, updateSignerProvider, updateAccount, connectionStatusRef])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
```

Comment:
We should only need to auto connect once when the page refreshes, so I've set this dependency list to empty, and it also auto connect once in wagmi.